cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowFunctionsTest.scala

```diff
@@ -28,63 +28,158 @@ class ShowFunctionsTest extends DocumentingTest {
       }
-      section("Listing functions with filtering on output columns") {
-        p(
-          """The listed functions can be filtered in multiple ways.
-            #One way is through the type keywords, `BUILT IN` and `USER DEFINED`.
-            #Another more flexible way is to use the `WHERE` clause.
-            #For example getting the name of all built-in functions starting on the letter 'a':""".stripMargin('#'))
-        query("SHOW BUILT IN FUNCTIONS YIELD name, isBuiltIn WHERE name STARTS WITH 'a'", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "isBuiltIn")
-          p.columnAs[String]("name").foreach(_ should startWith("a"))
-          p.columnAs[Boolean]("isBuiltIn").foreach(_ should be(true))
-        })) {
-          limitedResultTable(List("name", "isBuiltIn"), 15)
-        }
+    }
+    section("Listing functions with filtering on output columns") {
+      p(
+        """The listed functions can be filtered in multiple ways.
+          #One way is through the type keywords, `BUILT IN` and `USER DEFINED`.
+          #Another more flexible way is to use the `WHERE` clause.
+          #For example getting the name of all built-in functions starting on the letter 'a':""".stripMargin('#'))
+      query("""
+          #SHOW BUILT IN FUNCTIONS YIELD name, isBuiltIn
+          #WHERE name STARTS WITH 'a'""".stripMargin('#'),
+      ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "isBuiltIn")
+        p.columnAs[String]("name").foreach(_ should startWith("a"))
+        p.columnAs[Boolean]("isBuiltIn").foreach(_ should be(true))
+      })) {
+        limitedResultTable(List("name", "isBuiltIn"), 15)
+      }
+    }
+    section("Listing functions with other filtering") {
+      database("system")
+      initQueries(
+        "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED"
+      )
+      database("neo4j")
+      p(
+        """The listed functions can also be filtered on whether a user can execute them.
+          #This filtering is only available through the `EXECUTABLE` clause and not through the `WHERE` clause.
+          #This is due to using the users privileges instead of just filtering on the available output columns.""".stripMargin('#'))
+      p(
+        """There are two versions of the `EXECUTABLE` clause.
+          #The first one is to filter for the current user:""".stripMargin('#'))
+      login("jake", "abc123")
+      query("SHOW FUNCTIONS EXECUTABLE BY CURRENT USER YIELD *", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "category", "description", "signature", "isBuiltIn", "argumentDescription", "returnDescription", "aggregating", "rolesExecution", "rolesBoostedExecution")
+        p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
+        p.columnAs[List[String]]("rolesBoostedExecution").foreach(_ should be(null))
+      })) {
```

Comment:
All descriptions render as code blocks and within quotation marks. It's not strictly wrong, but it doesn't look so nice.